build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1551,6 +1557,10 @@ message BatchReadBlobsRequest {
 
   // The individual blob digests.
   repeated Digest digests = 2;
+
+  // A list of acceptable encodings to used for inlined data, in order of
```

Comment:
In decreasing order

Reply:
How does "in order of preference, from most to least preferred" sound?

Reply:
SGTM!

Reply:
Updated.